stingray/pulse/search.py

```diff
@@ -72,7 +81,7 @@ def _profile_fast(phase, nbin=128):
 
 
 def epoch_folding_search(times, frequencies, nbin=128, segment_size=5000,
-                         expocorr=False, gti=None, weights=1):
+                         expocorr=False, gti=None, weights=1, fdots=0):
```

Comment:
Remember to add new keyword arguments to the docstring, both here and further down below. :)

Reply:
Done